archive/scripting.html

```diff
@@ -1234,7 +1234,7 @@ <H3 ALIGN=Center>
     <LI>
       <A HREF="scripting-alpha.html#SetPref">SetPref(str,val[,val2])</A>
     <LI>
-      It it also possible to get the value of a preference item by preceding its
+      It is also possible to get the value of a preference item by preceding its
```

Comment:
Yes, exactly. Use the output of the 'tool' to direct attention towards *possible* problems, and start building changes upon evaluation.

Thus "had had" and "that that" are dropped as merely stylistic quirks. And other patterns emerge that are false positives. 

Use the tools, don't let the tools use you!

(Sorry, too many years at Wikipedia cleaning up after naïve editors' enthusiasms with 'tools'. Dozens and hundreds of clean up actions tell me tools have sharp edges, apparently unexpectedly)